src/main/scala/com/databricks/spark/xml/DefaultSource.scala

```diff
@@ -70,7 +70,8 @@ class DefaultSource
     XmlRelation(
       () => XmlFile.withCharset(sqlContext.sparkContext, path, charset, rowTag),
       Some(path),
-      parameters,
+      parameters + ("timezone" ->
```

Comment:
No need to pass this through. Just store it in TypeCast as a member

Reply:
I think I need a SparkSession to get `spark.sql.session.timeZone`. Is that ok to pass a SparkSession to TypeCast?

Reply:
I'd just use SparkSession.getActiveSession() to get the active session where you need it, no need to pass it

Reply:
I tried using this as well as SparkSession.getDefaultSession(), but it kept returning me null. I believe this is because the code is running on executors: https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala#L1073

Reply:
Oh I see, hm. OK, maybe this approach is the cleanest.